### DISCUSSION TITLE
How to cut individual atoms (water and ions) from a large trajectory (protein with solvent)?

### DISCUSSION THREAD

**Discussion by PinkFoxK** [TinkerTools/tinker#135] [Q&A] [answered]
There is a system - a protein dissolved in water (99268 atoms). We got a large trajectory for 700 GB and 1,750,000 frames. We wrote the trajectory in .DCD format (because trajectories in .ARC format are too heavy). There was a task to cut out water and ions from the trajectory file for further analysis. Has anyone faced a similar issue? Is it possible to cut atoms with tinker routines? Is the ARCEDIT subroutine suitable for this task, or is it intended only for working with the .ARC format? Are there any third-party (not from Tinker) programs for removing atoms from a trajectory in .DCD format?

**Comment by jayponder** [comment, marked as answer]
Hi, The ARCEDIT program will allow you to remove atoms from a DCD format file. Just read in the DCD file, followed by the corresponding Tinker XYZ file. Then use option 3 to remove specified ranges of atoms or specified ranges of frames. A new DCD format file will be written with the changes you requested. (Note when using the current version of Tinker, this new file will output with the ".arc" suffix, but it is actually in DCD format- this is, I suppose, a minor bug that we will fix in a future release.)

Note that before you can use this new DCD file, you will need to create a corresponding new Tinker XYZ file with the same changes you made in the DCD file. This step is necessary since DCD files contain only coordinates- they do not contain atom types, connectivity, or any other information. So whenever you try to use a DCD file, you must have the corresponding Tinker XYZ file.

**Reply by PinkFoxK** [reply]
Thanks for the answer!
How to specialize a range of atoms in the ARCEDIT program? What is the syntax for removing atoms from 2776 to 99268 in all frames of the trajectory file?

**Comment by jayponder** [comment]
Specifying atoms works in a similar way for all Tinker programs, including ARCEDIT. You can give individual atoms or ranges of atoms, where a negative atom number is the start of a range that ends with the following number. For example, 2 7 -11 16 -23 162 180 would select all the atoms 2, 7, 11-16, 23-162 and 180. So, in your case above, just use -2776 99268.
